AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/StorageEngineAdapter+SQLite.swift

```diff
@@ -188,22 +188,27 @@ final class SQLiteStorageEngineAdapter: StorageEngineAdapter {
 
     func query<M: Model>(_ modelType: M.Type,
                          predicate: QueryPredicate? = nil,
+                         sort: QuerySortInput? = nil,
                          paginationInput: QueryPaginationInput? = nil,
                          completion: DataStoreCallback<[M]>) {
         query(modelType,
               predicate: predicate,
+              sort: sort,
+              paginationInput: paginationInput,
```

Comment:
this looks kinda important, is this impacting customer calls? if so, may be worth calling out in a patch release and have in a separate PR

Reply:
Sure

Reply:
have u added issue to track this or opened PR? I'd say PR for this bug fix can probably go without additional test coverage (or a simple local store integration test that only uses pagination) since this PR adds the LocalStore integration tests that exercises the pagination with sort

Reply:
haven't yet. Not going to do so

Reply:
+1 and I would add we should fix it asap and also add a unit test to catch, since it seems to be a missing case.

Reply:
The unit tests for generating `paginationInput` SQLstatement is already there. 
Need to add an integration test and the PR https://github.com/aws-amplify/amplify-ios/pull/647 was merged